pkg/seccomp/filter.go

```diff
@@ -28,46 +28,66 @@ var (
 // BuildFilter does a basic validation for the provided seccomp profile
 // string and returns a filter for it.
 func BuildFilter(spec *specs.LinuxSeccomp) (*libseccomp.ScmpFilter, error) {
+	filter, profile, err := FilterFromSpec(spec)
+	if err != nil {
+		return nil, errors.Wrap(err, "build filter from runtime spec")
+	}
+
+	filter, err = addRuiles(filter, profile)
```

Comment:
Shouldn't this be addRules?

Reply:
Definitely, yes. I'll follow up on this with the next push :+1: